code/game/objects/items/weapons/cigs.dm

```diff
@@ -66,6 +66,10 @@ LIGHTERS ARE IN LIGHTERS.DM
 	..()
 	light()
 
+/obj/item/clothing/mask/cigarette/catch_fire()
+	if(!lit)
+		light("<span class='warning'>Your [name] is lit by the flames!</span>")
```

Comment:
At first glance this message would be displayed to [anyone on the turf](https://github.com/SabreML/Paradise/blob/Fire-Lighting/code/game/objects/items/weapons/cigs.dm#L164-L166). It doesn't make sense to put "You" for those who aren't wearing the cigarette.

Reply:
Ah, yep. That's a `visible_message`. One second.